packages/web-app-files/src/components/SideBar/Exif/ExifPanel.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div id="files-sidebar-panel-exif">
+    <dl class="exif-data-list">
+      <dt v-text="$gettext('Dimensions')" />
+      <dd data-testid="exif-panel-dimensions" v-text="dimensions" />
+      <dt v-text="$gettext('Device make')" />
+      <dd data-testid="exif-panel-cameraMake" v-text="cameraMake" />
```

Comment:
Why do we have the mix aus Kebap and camelCase here? 😅

Reply:
Because it's only data-testids? 🙈 Normally we use kebab case in html. I'm using them in a generic way and just inline the object keys. The object keys are camelCase. If you're super unhappy with that I can transform the object keys to kebab case first. 🤷

Reply:
fine for me, is wa just confused